dockstore-integration-testing/src/test/java/io/dockstore/client/cli/LimitedCRUDClientIT.java

```diff
@@ -254,13 +273,25 @@ public void testUploadZipHonorsVersionLimit() {
 
     }
 
-    private List<SourceFile> generateSourceFiles() throws IOException {
+    private List<SourceFile> generateSourceFiles(String lang) throws IOException {
+        String resourceFilePath;
+        String dockstorePath;
+        SourceFile.TypeEnum type;
+        if (lang.equals(CWL.getLowerShortName())) {
```

Comment:
Looks like:
- the input was originally an enum, then you converted it to lower case
- the if statement `CWL` is originally an enum, then you converted it to lower case

just don't convert?